go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/hashicorp/go-immutable-radix v1.3.1
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/golang-lru v0.5.4
-	github.com/in-toto/in-toto-golang v0.3.3
+	github.com/in-toto/in-toto-golang v0.3.4-0.20220709202702-fa494aaa0add
```

Comment:
Saw that they tagged new releases since this PR, so opened https://github.com/moby/buildkit/pull/3415